packages/js/src/plugins/auctionHouseModule/operations/depositToBuyerAccount.ts

```diff
@@ -59,7 +59,14 @@ export type DepositToBuyerAccountOperation = Operation<
  * @category Inputs
  */
 export type DepositToBuyerAccountInput = {
```

Comment:
This file is missing the comments on Operation and Transaction Builders.

Also, we need an empty line after the `auctionHouse` parameter.

Reply:
It's not, they were added while creating this operation.

Reply:
You're right I missed them. Sorry about that.